migrations/tables/api_task_status.yaml

```diff
@@ -1,4 +1,3 @@
-## no longer used, must keep for migrations to complete
```

Comment:
is this used now, or was this removed mistakenly?

Reply:
It is now used again.